summing up my work for first pull request

ebergeng opened the pull request.

ebergeng commented:
Added live server to package.json

cnnrbrn commented:
- I recommend moving all your styles out of index.scss and into partials. For example, the p, headings, a can go in a _typography.scss file, the body selector can go in a _bass.scss file.Button styles can go in a _buttons.scss file. Then in index.scss you just import all the files.
- If you want to publish your site on Netlify or a similar service rather link to the CDN verson of the Bootstrap JS file rather than the one in the node_modules folder. node_modules folder will not be present on a live site, only during development.
- The action on the form can be action="/profile" or "/feed", without the index.html part. The index.html page will be displayed from those folders as it is the default file for the folder so no need to include it in the url. It will keep the urls cleaner.
- You can link to the .css file with this href: "/dist/css/index.css" on every html page.
- Keep your image file names (like all file names and folders) in all lowercase letters.
- Not a big deal on this project but remember all img elements should have descriptive alt values for screen readers.

ebergeng commented:
1. moved my style out of index.scss and inn to there own style files and now importing them in the index.css
2. Changed the the script link to the CDN verson.
3. Changed the action on the form to be /feed insted of /feed/index.html
4. Changed the img file name to be all lowercase
5. Added alt text to img.
